spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/TaskTemplate.java

```diff
@@ -231,6 +237,13 @@ public String taskExecutionLog(String externalExecutionId, String platform) {
 		return restTemplate.getForObject(retrieveLogLink.expand(map).getHref(), String.class);
 	}
 
+	@Override
+	public String taskExecutionLog(Long executionId) {
+		Map<String,Long> map = new HashMap<>();
```

Comment:
**[TinyNit]** (I am sure IntelliJ will be whining about the same on commit analyze, but...) `Collections.singletonMap` can be handy in these cases. Wdyt?

Reply:
Good catch!